src/internal-version.js

```diff
@@ -24,5 +24,5 @@
  * @return {string}
  */
 export function internalRuntimeVersion() {
-  return '$internalRuntimeVersion$';
+  return INTERNAL_RUNTIME_VERSION; // eslint-disable-line no-undef
```

Comment:
note: I could also add this to the eslint config, but wasn't sure if that were preferable

Reply:
I think the most unified solution, in keeping with the all-constants-in-one-file angle, would be to have a similar single forbidden-terms rule that uses one regex to test for `/INTERNAL_RUNTIME_VERSION|IS_(FORTESTING|ESM|SXG|MINIFIED)/` and has a very short list of these files that use it (and maybe this particular file gets moved into `#core/mode` after this PR). That way if we ever need to adjust build constants we can easily find a list of files that can use them.